src/OrchardCore/OrchardCore.Users.Core/Models/TwoFactorLoginSettings.cs

```diff
@@ -7,4 +7,6 @@ public class TwoFactorLoginSettings
     public bool AllowRememberClientTwoFactorAuthentication { get; set; }
 
     public int NumberOfRecoveryCodesToGenerate { get; set; } = 5;
+
+    public bool UseSiteTheme { get; set; }
```

Comment:
The phone icon isn't loading with the Blog theme, see bottom:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/ba26445d-8aa4-4992-8679-91bc482abe82)

Not sure if this needs to be fixed in this module though.

Reply:
Probably just need to add font awesome in the TeoFactor layout

Reply:
I added the font awesome.